apps/daimo-mobile/eas.json

```diff
@@ -24,7 +25,8 @@
         "gradleCommand": ":app:assembleDebug"
       },
       "ios": {
-        "buildConfiguration": "Debug"
+        "buildConfiguration": "Debug",
+        "supportsTablet": true
```

Comment:
i think we have to set this in `app.config.js`, not in eas.json

<img width="456" alt="image" src="https://github.com/daimo-eth/daimo/assets/169280/0352b740-880b-4675-8775-3767159ebc1d">

Reply:
https://docs.expo.dev/versions/latest/config/app/#supportstablet